fili-core/src/test/groovy/com/yahoo/bard/webservice/data/metric/TemplateDruidQuerySpec.groovy

```diff
@@ -469,4 +470,45 @@ class TemplateDruidQuerySpec extends Specification {
                 .find { it -> it.getName() == arithmeticPostAggFromAliasName}
         resultArithmeticPa.getPostAggregations() == [arithmeticPostAggOperand1, resultAliasPa] as List
     }
+
+    def "Simple tests by passing static dimensions while building TDQ"() {
```

Comment:
test name should somewhat state what the outcome is.  
So something like 'Dimensions are retained when tdqs are merged'
Also, I'd suggest doing this with a where block where you test the following cases:
leftside = [dim1], rightside=[]
leftside=[dim1], rightside[dim2]
leftside=[dim1,dim2], rightside[dim1]
Just to make sure merging does what we want it to do with one sideded and overlapping dimension sets.

Reply:
If you do do 'where' then the you should put variables in the method name with the list of dimensions being passed, e.g. "When #dimensions1 merged with #dimension2 the resulting dimension set is #expected"